cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -379,11 +459,21 @@ struct weak_ordering_comparator_impl {
  * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
  */
 template <typename Comparator>
-using less_comparator = weak_ordering_comparator_impl<Comparator, weak_ordering::LESS>;
+struct less_comparator : weak_ordering_comparator_impl<Comparator, weak_ordering::LESS> {
+  less_comparator(Comparator const& c)
```

Comment:
For consistency with `weak_ordering_comparator_impl` and other places where a comparator is passed, I'd like to see this renamed:
```suggestion
  less_comparator(Comparator const& comparator)
```